quipucords/quipucords
# Add facts about how many users are on a system

Closes #2009 

To test: 
1. Run a network scan
2. See that the new facts exist for both details/deployments reports; `system_user_count` `user_login_history`. 

***_Note_***: 
-  In order to check that login is working - you should login to one of the systems that you are scanning as a different user than the one that you scan with and make sure that it is in the history. 
 - To check the number of users - you can also login to a system and add users/manually inspect the /etc/passwd file and make sure that the number of real users matches the number reported.
